packages/kbn-cli-dev-mode/src/get_server_watch_paths.ts

```diff
@@ -52,7 +52,7 @@ export function getServerWatchPaths({ pluginPaths, pluginScanDirs }: Options) {
 
   const ignorePaths = [
     /[\\\/](\..*|node_modules|bower_components|target|public|__[a-z0-9_]+__|coverage)([\\\/]|$)/,
-    /\.test\.(js|tsx?)$/,
+    /\.(test|spec)\.(js|ts|tsx?)$/,
```

Comment:
tsx? Covers both ts and tsx files, but feel free to go with a more obvious `ts|tsx`